app/models/content_store/content_item.rb

```diff
@@ -0,0 +1,54 @@
+class ContentItem
+  attr_reader :edition
+
+  def initialize(edition)
+    @edition = edition
+  end
+
+  def present
+    SymbolizeJSON.symbolize(
+      edition.attributes.merge(
+        api_path: api_path,
+        api_url: api_url,
+        web_url: web_url,
+        withdrawn: withdrawn?,
+        content_id: content_id,
+        locale: locale,
+      )
+    )
+  end
+
+  def document_type
+    edition.document_type
+  end
+
+  def content_id
+    edition.document.content_id
+  end
+
+private
+
+  def locale
+    edition.document.locale
+  end
+
+  def withdrawn?
+    edition.unpublishing.present? && edition.unpublishing.withdrawal?
```

Comment:
I think we can just use `edition.withdrawn?` etc for these methods and the ones below. Or is the plan to remove those methods from `Edition` at some point?